attestation-service/config.json

```diff
@@ -1,12 +1,11 @@
 {
     "work_dir": "/var/lib/attestation-service/",
-    "policy_engine": "opa",
     "rvps_config": {
         "store_type": "LocalFs",
         "remote_addr": ""
     },
-    "attestation_token_broker": "Simple",
-    "attestation_token_config": {
+    "attestation_token_broker": {
+        "type": "Simple",
```

Comment:
Do we need to mark this as `Ear` to align with what we changed in `README.md`? s.t.
 > If the policy is not updated, the AS will use the [default policy](src/token/ear_default_policy.rego).